src/internal/storage/chunk/client.go

```diff
@@ -43,73 +50,105 @@ func NewClient(store kv.Store, mdstore MetadataStore, tr track.Tracker, name str
 
 // Create creates a new chunk from metadata and chunkData.
 // It returns the ID for the chunk
-func (c *Client) Create(ctx context.Context, md Metadata, chunkData []byte) (_ ID, retErr error) {
+func (c *trackedClient) Create(ctx context.Context, md Metadata, chunkData []byte) (_ ID, retErr error) {
+	if c.renewer == nil {
+		panic("client must be named to create chunks")
+	}
 	chunkID := Hash(chunkData)
 	var pointsTo []string
 	for _, cid := range md.PointsTo {
-		pointsTo = append(pointsTo, ObjectID(cid))
+		pointsTo = append(pointsTo, cid.TrackerID())
 	}
-	chunkOID := ObjectID(chunkID)
-	if err := dbutil.WithTx(ctx, c.mdstore.DB(), func(tx *sqlx.Tx) error {
-		if err := c.mdstore.SetTx(tx, chunkID, md); err != nil && err != ErrMetadataExists {
+	chunkOID := chunkID.TrackerID()
```

Comment:
We should probably use `TID` as the suffix for tracker ID to avoid confusion between object storage objects and tracker objects. This is pretty minor, but I think it might be important for readability.

Reply:
Yeah this makes sense.